tutorials/model_creation.md

```diff
@@ -0,0 +1,92 @@
+\page model_creation Model creation
+
+This tutorial gives an introduction to the Ignition Gazebo's service `/world/<world name>/create`.
+This service will allow to create models in the scene such us spheres, lights, etc.
+
+Ignition Gazebo creates many services. You can check this typing:
```

Comment:
This could be a bit misleading, because some of these services are only created if certain plugins are loaded. I think it would be good to mention that the `UserCommands` plugin needs to be loaded to offer the `create` service.